test/e2e/suites/unmanaged/unmanaged_functional_test.go

```diff
@@ -862,6 +722,37 @@ var _ = ginkgo.Context("[unmanaged] [functional]", func() {
 				AvailabilityZone:  "us-west-2a",
 			}, e2eCtx)
 			wlClusterInfra.CreateInfrastructure()
+
+			ginkgo.By("Creating VPC peerings")
+			cPeering, _ = shared.CreatePeering(e2eCtx, mgmtClusterName+"-"+wlClusterName, *mgmtClusterInfra.VPC.VpcId, *wlClusterInfra.VPC.VpcId)
+		})
+
+		ginkgo.JustAfterEach(func() {
+			shared.ReleaseResources(requiredResources, config.GinkgoConfig.ParallelNode, flock.New(shared.ResourceQuotaFilePath))
+			shared.DumpSpecResourcesAndCleanup(ctx, "", namespace, e2eCtx)
```

Comment:
If there is a problem with calling this upon failure here, this might be a problem in other tests as well, right?